### PR TITLE
more option provided for the disk format encryption scenario.

### PR DESCRIPTION
**Pull request by andyliuliming**
--------------------MESSAGE FROM ADMIN, DELETE BEFORE SUBMITTING----------------------

Thank you for deciding to contribute to this repository, we're thrilled to have you contribute a template or a fix! :tada: Before we proceed, please be sure to review the contribution guidelines that will make the review process go smoother.
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

We've created an outline of recommended sections to fill out that will help make this Pull Request awesome!

--------------------MESSAGE FROM ADMIN, DELETE BEFORE SUBMITTING----------------------

### Changelog
*
*
*
*
*

### Description of the change

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@andyliuliming__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (andliu). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: 201-encrypt-running-linux-vm/azuredeploy.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "diskFormatQuery": {
       "defaultValue": "",
       "metadata": {
-        "description": "the query string used to identify the disks to format and encrypt. This parameter only works when you choose the EncryptionOperation as EnableEncryptionFormat.[{\"scsi\":\"6:0:0:1\",\"name\":\"azuredisk\"},{\"scsi\":\"6:0:0:1\",\"name\":\"azuredisk2\"}], the scsi means the disk scsi number, the name means the mount point name"
+        "description": "the query string used to identify the disks to format and encrypt. This parameter only works when you choose the EncryptionOperation as EnableEncryptionFormat.[{\"scsi\":\"6:0:0:1\",\"name\":\"azuredisk\"},{\"dev_path\":\"/dev/sdx\",\"name\":\"azuredisk2\",\"file_system\":\"ext4\"}], the scsi means the disk scsi number, the name means the mount point name, the file_system is the file system to use."
```
Comment (singhkay) [inline 201-encrypt-running-linux-vm/azuredeploy.json]: Can you describe what this change does in the PR description?

**Comment by singhkay** [comment]
@SudhakaraReddyEvuri Can you review this?

**Comment by SudhakaraReddyEvuri** [comment]
@singhkay  Changes look good to me. @andyliuliming Andy, did you test the updated templates?  I see the Travis is failing.

**Comment by singhkay** [comment]
Thanks @SudhakaraReddyEvuri. Failure is expected since this PR requires previous setup. merging

**Pull request merged**